modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerAbstractTest.java

```diff
@@ -28,6 +30,7 @@
 import java.util.logging.Handler;
 import java.util.logging.Logger;
 import java.util.stream.Collectors;
+
```

Comment:
Empty line not needed there https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-PackageImporting